jekyll/_docs/install-and-use-yarn.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: classic-docs
+title: "Install and Use Yarn (the NPM replacement) on CircleCI"
+short-title: "Install and Use Yarn on CircleCI"
+categories: [how-to]
+description: "How to modify `circle.yml` in order to install and use Yarn on CircleCI."
+---
+
+<img src="{{site.baseurl}}/assets/img/logos/yarn-logo.svg" style="display:block;margin:15px auto;width:40%;min-width:320px;" alt="Yarn Logo" />
+
+[Yarn][yarn-site] is an open-source package manager for JavaScript. Both Yarn itself as well as the packages it installs can be cached on CircleCI. This can potentially speed up builds but more importantly can remove network connectivity as a source of problems.
```

Comment:
I don't know enough about Yarn to know if this is true, but it's a pretty strong statement to say that Yarn removes network connectivity _entirely_ as a potential problem source. If that really is true, I would love more explanation on why that's the case.